mne/io/brainvision/brainvision.py

```diff
@@ -956,7 +968,8 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale, montage):
 
     info._update_redundant()
     info._check_consistency()
-    return info, data_filename, fmt, order, mrk_fname, montage, n_samples
+    return (info, data_fname, fmt, order, n_samples, mrk_fname, montage,
+            orig_units)
```

Comment:
why did the order of the parameters change?

Reply:
because I corrected a copy paste error and changed some names in the process, see https://github.com/mne-tools/mne-python/pull/5633/files#r226644804

Reply:
+1